### ISSUE TITLE
podStation lost list of podcasts (stations).

### ISSUE THREAD

**Issue by dellagustin** [podStation/podStation#59]
I have been contacted by e-mail and through chrome web store by two different users reporting the same issue: podStation is now empty (no list of podcasts, or stations as some call them).

One of the users reported being using Linux Mint with Google Chrome.

If you are also having this issue, please react to this issue with a 👍

**Comment by dellagustin** [comment]
Dear user,

If you had this same issue, please help me investigated by doing the following steps:

1. Open podStation
2. Right click anywhere in the podStation page and select _Inspect_ on the menu
3. On the Dev Tools window, select Console:
    ![image](https://user-images.githubusercontent.com/260113/73594323-967eb880-450d-11ea-9774-a9a79f240411.png)
4. On the console, copy and paste the following piece of code and press _Enter_: `chrome.storage.sync.get((s) => console.log(JSON.stringify(s)));`
5. Copy the result and send it to me by email: podstationapp@gmail.com
6. Optionally, do the same with the following piece of code: `chrome.storage.local.get((s) => console.log(JSON.stringify(s)));`

**Important!** By doing so you will be sending me the list of podcasts you subscribe to - please do not send it unless you are comfortable sharing this information.

**Comment by dellagustin** [comment]
One user reported the following result:
```
VM197:1 Uncaught TypeError: Cannot read property 'get' of undefined
    at <anonymous>:1:22
```

This is very strange. I was expecting either:
- a full storage object (which would not explain why it is not working)
- an empty storage object (still does not explain the data loss)
- `VM95:1 Uncaught TypeError: Cannot read property 'sync' of undefined
    at <anonymous>:1:16` - in case the script was executed on the wrong console (not the extension)

But the error the user received would only occur is chome.storage is an object, but sync is undefined, which I can't explain.

**Comment by dellagustin** [comment]
May be of use, I found out that on windows there is some logging about the sync storage at the folder `C:\Users\<user>\AppData\Local\Google\Chrome\User Data\Default\Sync Extension Settings\bpcagekijmfcocgjlnnhpdogbplajjfn`

There is a `.log` file.  
It is a binary, but there is a lot of identifiable JSON content there that could maybe be used to recover user sync data.

**Comment by dellagustin** [comment]
From one of the users I got the following information:
> By the way I can't add any podcasts to podStation.

And after that, the following error from the console:

```
Uncaught TypeError: Cannot read property 'setActionHandler' of undefined
    at Object.audioPlayerService (audioPlayer.js:547)
    at Object.invoke (angular.js:4478)
    at Object.enforcedReturnValue [as $get] (angular.js:4330)
    at Object.invoke (angular.js:4478)
    at angular.js:4295
    at getService (angular.js:4437)
    at Object.invoke (angular.js:4469)
    at angular.js:4299
    at forEach (angular.js:336)
    at createInjector (angular.js:4299)
reuse.js:23 Uncaught Error: could not get service "messageService", inke
    at getAngularService (reuse.js:23)
    at getMessageService (reuse.js:30)
    at registerMessageListeners (podcastManager.js:448)
    at HTMLDocument.<anonymous> (podcastManager.js:442)
    at j (jquery.min.js:sourcemap:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:sourcemap:2)
    at Function.ready (jquery.min.js:sourcemap:2)
    at HTMLDocument.I (jquery.min.js:sourcemap:2)
6background.html:1 Error in event handler for runtime.onMessage: TypeError: Cannot read property 'paused' of undefined
    at buildAudioInfo (chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/background/entities/audioPlayer.js:205:26)
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/background/entities/audioPlayer.js:509:17
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:42:19
    at dispatchToListeners (chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:41:33)
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:89:12
```

This was the same symptom from #63 .

Inspecting the information from the bug report on chrome webstore left by another user, I noticed that the user was using podStation at Chrome Chrome/68.0.3440.75.

It happens that chrome (Desktop) only support MediaSession as of version 73:
https://developer.mozilla.org/en-US/docs/Web/API/MediaSession

I managed to fix easily with d5a9bda30d00921a6461b1204e1f03dfc35110f7.

Version 1.22.1 is released, still pending review from Chrome Webstore, as soon as it is published podStation should work again on Opera.
